app/models/user.rb

```diff
@@ -145,6 +145,7 @@ def self.generate_csv_list(user_list)
   end
 
   def self.upload_user_list(user_class, user_list, encoding)
+    max_invalid_lines = 10
```

Comment:
Wondering if we should hard code this, or if it should be configurable somewhere? Not sure how it all works.